### PR TITLE
Created variations for Slate

### PR DESCRIPTION
**Pull request by vistajess**

### CODE REVIEW

File: scripts/components/Dropdown/index.jsx [hunk 1]
```diff
@@ -67,8 +67,7 @@ class Dropdown extends React.Component {
       ? DOM[element]
       : element)(
         props,
-        trigger(TRIGGER_REF)
-        active ? children : null
+        trigger(TRIGGER_REF) === active ? children : null
```
Comment (srph) [inline scripts/components/Dropdown/index.jsx]: ?

File: styles/components/slate.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		margin: 0px auto;
+	}
+	&.half-slate {
+		width: 50%;
+	}
+	&.tidy-background {
+		background : $brand-white;
+		border-radius: 0px;
+	}
+}
```
Comment (srph) [inline styles/components/slate.scss:20]: Where are the indents? :smile:

**Comment by srph** [comment]
Remove all build scripts.
- Go to root (`/path/to/june`)
- Run `rm dist/*.js`

Mind adding a screenshot?

File: Gemfile.lock [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+PLATFORMS
+  x86-mingw32
+
+DEPENDENCIES
+  listen (~> 2.8)
+  sass
+  sass-css-importer
+
+BUNDLED WITH
+   1.10.6
```
Comment (srph) [inline Gemfile.lock]: Mind removing this file for now? `rm Gemfile.lock`

File: scripts/components/Dropdown/index.jsx (hunk 1 above)
Comment (vistajess) [inline scripts/components/Dropdown/index.jsx]: Si bryan naglagay

**Comment by vistajess** [comment]
@srph Review :)

File: scripts/components/Dropdown/index.jsx (hunk 1 above)
Comment (srph) [inline scripts/components/Dropdown/index.jsx]: stash mo nlng tong file na to

**Comment by vistajess** [comment]
slate

File: dist/components/Dropdown/index.js [hunk 4]
```diff
@@ -0,0 +1,148 @@
+'use strict';
```
Comment (srph) [inline dist/components/Dropdown/index.js]: Can you remove this file?

**Pull request merged**